quinn-proto/src/connection.rs

```diff
@@ -2666,6 +2669,27 @@ impl Connection {
     }
 }
 
+enum CryptoLevel {
+    Initial,
+    OneRtt,
+}
+
+impl CryptoLevel {
+    fn is_initial(&self) -> bool {
+        match *self {
+            CryptoLevel::Initial => true,
+            _ => false,
+        }
+    }
+
+    fn is_1rtt(&self) -> bool {
+        match *self {
+            CryptoLevel::OneRtt => true,
+            _ => false,
+        }
+    }
+}
```

Comment:
Given that this enum likely won't ever contain data, these methods should be removed in favor of applying `#[derive(Eq, PartialEq)]` to the enum and inspecting values with `==`/`!=`.

Reply:
This is what I was looking for, thank you.